### PR TITLE
[V4] make lastname optional

### PR DESCRIPTION
**Pull request by vincentbpro**
### What does it do?

Makes the lastname optional for admin users

#### Front-end
Lastname field made optional at:
- register first admin form
- create admin user form
- update admin user form
- user profile form (admin/me)

Updated references to lastname across the app to handle an empty string

#### Back-end

Updated validations to accept empty lastname

### Why is it needed?

To reduce friction during the first admin registration

### CODE REVIEW

File: packages/core/admin/admin/src/utils/getFullName.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/**
+ * Get fullname from firstname+lastname joined with a space
+ * Handle missing lastname
+ * @param {string} firstname
+ * @param {string} lastname
+ */
+export default (firstname, lastname = '') => {
+  return [firstname, lastname].filter(e => e !== '').join(' ');
+};
```
Comment (HichamELBSI) [inline packages/core/admin/admin/src/utils/getFullName.js:9]: Can you add some test for this file please

**Review by HichamELBSI — CHANGES_REQUESTED** [review]

**Review by alexandrebodin — COMMENTED** [review]
Overall looks good 👍

**Review by HichamELBSI — APPROVED** [review]
LGTM

**Comment by alexandrebodin** [comment]
just a conflict in a test file apparently

**Pull request merged**